src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -0,0 +1,70 @@
+package dataloader;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.csv.CsvReadOptions;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.TreeMap;
+
+
+/**
+ * Class that takes a String as path file
+ * to csv files
+ * and creates datasets in two formats
+ * TreeMap & Table from tablesaw.
+ */
+
+public class CsvDataLoader {
+
+    /**
+     * Simple method that parses data set from a csv file
+     * without knowing the headers of the file
+     */
+    public void parseFile(String csvFile, TreeMap dataSet) throws IOException {
+
+        Reader csvData = new FileReader(csvFile);
+
+        CSVParser parser = CSVParser.parse(csvData, CSVFormat.DEFAULT);
+
+        for (CSVRecord record : parser) {
+
+            for (String field : record) {
+                dataSet.keySet();
+                dataSet.get(field);
+
+                System.out.println("\"" + field + "\", ");
+            }
+        }
+    }
+
+    /**
+     * Reads from csv in file system with columns separated by commas
+     * and the file has a header row.
+     * Missing values are treated with an indicator.
+     */
+    public Table parseFile(String csvFile, Table dataSet) throws IOException {
```

Comment:
Note that the implementations are similar so we should have one implemented by the other. Also what happens when there are more user options we need to take into account?

Reply:
we have many option to take into account. like taking data from an online source, etc.
I think we should make some tests, gather all the possible inputs of data that we might think of and then decide which ones to incorporate.

Reply:
should like a good idea but I think we should assign this to another PR...can you add the tab in the TODO column of the Kanban board?

Reply:
perhaps we should consider making dataSet a local variable. what do you think?

Reply:
I agree checkout this PR: https://github.com/jethronap/JavaDataAnalysis/pull/8/files

Reply:
should we implement that idea then?

Reply:
yes

Reply:
did that.